packages/e2e-tests/fixtures/blocks/core__cover__deprecated-1.serialized.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:cover {"url":"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==","id":34,"className":"wp-block-cover-image"} -->
```

Comment:
It doesn't look it gets converted properly.

Reply:
Managed to figure this one out as well. Had to add:
```
		supports: {
			className: false,
		},
```
So that the change in class name from `wp-block-cover-image` to `wp-block-cover` works correctly.

Also added a migration function so that the inner text content is converted to a paragraph block.